tests/functional/lib/helpers.js

```diff
@@ -792,24 +792,22 @@ define([
     };
   }
 
-  function fillOutCompleteResetPassword(context, password, vpassword) {
-    return getRemote(context)
+  /**
+   * Fill out and submit  the complete reset password form
+   * @param {String} password - new password
+   * @param {String} vpassword - new verification password
+   * @returns {promise}
+   */
+  function fillOutCompleteResetPassword(password, vpassword) {
+    return function () {
```

Comment:
Once this whole series has merged, I'm going to open another PR to `thenify` all of the helper functions. I'm not doing it as I go along to avoid adding more inconsistencies.